File: src/main/java/com/github/wgzhao/dbquery/entities/QueryConfigSign.java
package com.github.wgzhao.dbquery.entities;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name = "query_config_sign")
@Setter
@Getter
public class QueryConfigSign {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private String selectId;

    private String appId;
}

File: src/main/kotlin/com/github/wgzhao/dbquery/entities/QueryConfigSign.kt
package com.github.wgzhao.dbquery.entities

import jakarta.persistence.*
import lombok.Getter
import lombok.Setter

@Entity
@Table(name = "query_config_sign")

data class QueryConfigSign (
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private val id: Long? = null,
    private val selectId: String? = null,
    private val appId: String? = null
)